### ISSUE TITLE
Demo link

### ISSUE THREAD

**Issue by brunolemos** [lesniakania/react-native-draggable-board#3]
Could you provide a demo link so we can try it without having to clone and build?
Exponent is great for that. https://getexponent.com

**Comment by lesniakania** [comment]
Hi. Good suggestion, thanks.

**Comment by lesniakania** [comment]
@brunolemos hm, actually I'm not sure if I don't something, or it's just not possible.
I wanted to use this getexponent.com via rnplay.org, but I can't use external libraries (only react native modules) there.

**Comment by brunolemos** [comment]
Apparently you don't have any native module dependency in your package, so you should be able to use exponent. (not rnplay)
I think you have to download their XDE and compile your app there, but I'm not sure, never done it.
@brentvatne can help.

**Comment by lesniakania** [comment]
I see, I will investigate it later, but any tips are welcome, as I haven't done it before ;]

**Comment by brentvatne** [comment]
yeah, it should work with https://expo.io/ no problem! also, sketch.expo.io is the successor to rnplay, and will support adding npm packages soon
